### ISSUE TITLE
PEtab documentation

### ISSUE THREAD

**Issue by thomassligon** [PEtab-dev/PEtab#317]
documentation refers to:
[https://github.com/PEtab-dev/PEtab/blob/master/doc/documentation_data_format.md](https://github.com/PEtab-dev/PEtab/blob/master/doc/documentation_data_format.md)

row or column?
General Remarks: Fields in "[]" in the second **row** are optional and may be left empty.
Condition table has “[conditionName]” in the first **row**, second **column**.
What does “[]” mean? Should the general remark apply to rows or columns?
Could it be that rows and columns are reversed when the .tsv is read into python?
Same for Measurement table “[preequilibrationConditionId]”, “[observableParameters]”, “[noiseParameters]”, “[datasetId]”, “[replicateId]”.
Same for Observables table “[observableTransformation]”.
Same for Parameter table “[parameterName]”.

Yaml error:
Provided YAML file does not adhere to PEtab schema: 'format_version' is a required property
Need: 
`format_version: 1`
Not described in:
[https://petab.readthedocs.io/en/latest/example/example_petablint.html](https://petab.readthedocs.io/en/latest/example/example_petablint.html)
but the lines from the Bachmann sample show how to do it, and it is in the schema.

**Comment by yannikschaelte** [comment]
@thomassligon not sure if I understand what you mean here. could you explain?

**Comment by thomassligon** [comment]
Hi Yannik, I would be happy to explain, but I'm not sure that more prose is the best way to do it. Do you have some time when we could discuss this on the phone or Skype or something else?

**Comment by thomassligon** [comment]
Hi Yannik,
I haven't seen a response to my suggestion for a discussion, so I will try to do it here. Unfortunately, making things very explicit will involve a lot of text.
The general remark is:
`Fields in "[]" in the second row are optional and may be left empty.`
In the condition table, the second row contains
`conditionId1<tab>conditionName1<tab>NUMERIC\<tab>parameterId\<tab>stateId\<tab>compartmentId<tab>...<tab>...
`
so there is no "[]" and no optional field, but conditionName1 might be optional, so maybe it should be "[conditionName1]". The second column consists of
`[conditionName]<tab>conditionName1`
so it seems logical that the whole column might be optional. In the measurement table, the second row consists of
`observableId<tab>[conditionId]<tab>conditionId<tab>NUMERIC<tab>NUMERIC\<tab>'inf'`
so the general remark says that [conditionId] is optional. In fact, that is true for "the second row and all following rows". In addition, the whole second column is presumably optional.
In summary, and after looking at this again, it looks to me like rows and columns are not reversed, but the remark
`Fields in "[]" in the second row`
should say
`Fields in "[]" in the second row and all following rows`
and
`Columns containing in "[]" in the first row are optional.`
Thanks for your patience.

**Comment by yannikschaelte** [comment]
Hi Tom, thanks for pointing this out. This should be fixed in #378.

**Comment by JanHasenauer** [comment]
@thomassligon Can you confirm that this has been fixed? If so, please close the issue.
